README.md

```diff
@@ -180,10 +178,13 @@ Deploy the compiled component with the contracts from the previous steps. Review
 ```bash docci-delay-per-cmd=1
 # Build your service JSON
 # The trigger is on `local2` while the submission is on `local` (main)
-SUBMIT_CHAIN=local SUBMIT_ADDRESS=$SERVICE_HANDLER TRIGGER_ADDRESS=$SERVICE_TRIGGER_ADDR TRIGGER_CHAIN=local2 COMPONENT_FILENAME=eth_price_oracle.wasm sh ./script/build_service.sh
+export TRIGGER_EVENT="0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef" # found in events tab, topic0 of base etherscan
```

Comment:
Only needed for the wETH contract transfer event (purely hacky so i did not have to upload a contract). Remove this if you use the default NewTrigger(bytes) from the SimpleTrigger.sol example